dg-commons/pom.xml

```diff
@@ -10,20 +10,123 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>dg-commons</artifactId>
+    <groupId>org.finra.datagenerator.commons</groupId>
+    <version>2.2-SNAPSHOT</version>
     <name>Data Generator Common Utils</name>
 
     <dependencies>
+        <!-- Used for base-64 encoding for HTTP basic authentication (which we need to access Jenkins). -->
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.10</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.7.5</version>
+        </dependency>
+        <dependency>
```

Comment:
Jsch is used by JschHelper. GOATS uses these helper functions to do
automated testing over SSH.

On Tuesday, July 21, 2015, Mohamed Ibrahim <notifications@github.com> wrote:

> In dg-commons/pom.xml
> <https://github.com/FINRAOS/DataGenerator/pull/222#discussion_r35176279>:
>
> >      <name>Data Generator Common Utils</name>
> >
> >      <dependencies>
> > +        <!-- Used for base-64 encoding for HTTP basic authentication (which we need to access Jenkins). -->
> > +        <dependency>
> > +            <groupId>commons-codec</groupId>
> > +            <artifactId>commons-codec</artifactId>
> > +            <version>1.10</version>
> > +        </dependency>
> > +        <dependency>
> > +            <groupId>org.slf4j</groupId>
> > +            <artifactId>slf4j-log4j12</artifactId>
> > +            <version>1.7.5</version>
> > +        </dependency>
> > +        <dependency>
>
> Why do we need this ? that's an SSH library ( http://www.jcraft.com/jsch/
> )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FINRAOS/DataGenerator/pull/222/files#r35176279>.
>